packages/babel-plugin-minify-type-constructors/__tests__/fixtures/basic/expected.js

```diff
@@ -0,0 +1,5 @@
+!!x;
++x;
+x + "";
```

Comment:
Shouldn’t this be `x+""` (i.e. no spaces around the `+`)?

```js
x+""
```

Reply:
This is taken care of by babel-generator's `minify:true` mode.